similarity/manage.py

```diff
@@ -0,0 +1,147 @@
+from __future__ import print_function
+from flask.cli import FlaskGroup
+import click
+
+import webserver
+import metrics
+from operations import HybridMetric
+import db
+
+from sqlalchemy import text
+
+PROCESS_BATCH_SIZE = 10000
+
+cli = FlaskGroup(add_default_commands=False, create_app=webserver.create_app_flaskgroup)
+
+
+@cli.command()
+def init_similarity():
+    click.echo('Copying data')
+    with db.engine.begin() as connection:
+        query = text("""
+        INSERT INTO similarity(id)
+             SELECT id
+               FROM lowlevel
+        """)
+        connection.execute(query)
+
+
+def _get_recordings_without_similarity(connection, name, batch_size):
+    query = text("""
+        SELECT id
+          FROM similarity
+         WHERE %(metric)s IS NULL
+         LIMIT %(limit)s
+    """ % {'metric': name, 'limit': batch_size})
+    result = connection.execute(query)
+    rows = result.fetchall()
+    if not rows:
+        return []
+    ids = zip(*rows)[0]
```

Comment:
this should be `[r[‘id’] for r in rows]` for readability

Reply:
and it also means we don’t need the `if not rows` check

Reply:
I wrote this query in a different way, so I was able to remove this function entirely. I mainly altered it so we could add row by row, with all the metrics at the same time. Doing it that way, it didn't make sense to query for recordings like this anymore (plus, we set the metrics columns as not nullable).